tests/util/transactions.ts

```diff
@@ -131,3 +133,67 @@ export async function createContractExecution(
 
   return tx;
 }
+
+// The parameters passed to the function are assumed to have all been converted to hexa
```

Comment:
```suggestion
// The parameters passed to the function are assumed to have all been converted to hexadecimal
```
unless there is an abbreviation I'm not aware of